UndertaleModLib/UndertaleData.cs

```diff
@@ -490,19 +485,38 @@ private bool TestGMS1Version(uint stableBuild, uint betaBuild, bool allowGMS2 =
         /// <param name="release">The release version.</param>
         /// <param name="build">The build version.</param>
         /// <returns>Whether the version of the data file is the same or higher than a specified version.</returns>
-        public bool IsVersionAtLeast(uint major, uint minor, uint release, uint build)
+        public bool IsVersionAtLeast(uint major, uint minor = 0, uint release = 0, uint build = 0)
         {
-            if (GeneralInfo.Major != major)
-                return (GeneralInfo.Major > major);
+            uint gameMajor = GeneralInfo.Major;
+            uint gameMinor = GeneralInfo.Minor;
+            uint gameRelease = GeneralInfo.Release;
+            uint gameBuild = GeneralInfo.Build;
+            switch (GeneralInfo.GMS2Version)
```

Comment:
Small comment for this would be nice, something along the lines of "if gms2version is set, we readjust major/minor/release/build based on those."

Reply:
Added "If GMS2Version is set, we increase the version values to match."